src/treeffuser/_base_tabular_diffusion.py

```diff
@@ -23,6 +24,31 @@
 from treeffuser.sde import DiffusionSDE
 from treeffuser.sde import sdeint
 
+###################################################
+# Helper functions
+###################################################
+
+
+def _check_array(array: ndarray[float]):
+    if not isinstance(array, np.ndarray):
+        raise TypeError("Input must be a numpy array.")
```

Comment:
Could be nice to issue a warning of float is not 32

Reply:
added